x11/auto-multiple-choice/Portfile

```diff
@@ -27,27 +27,26 @@ subport auto-multiple-choice-devel {}
 
 if {${subport} eq ${name}} {
     # release
-    set amc.version.main        1.5.1
+    set amc.version.main        1.5.2
+    set amc.version.secondary   20221031112254
     version                 ${amc.version.main}
-    revision                3
-    checksums               rmd160  e7c260b0ff2f93e70a2e27ec7a33e68c89622805 \
-                            sha256  329b2a0c8f9ad2a8219865acc79ac0441ef7c7179c19b1cb4795fb2186a4760e \
-                            size    11251717
+    checksums               rmd160  76a1b398c30a1f70a79fd069397af39522e8d5f9 \
+                            sha256  4087a8fefd184fa325af958980ce36d03c9a04f14912e056da12b2202c33b338 \
+                            size    13107480
 
     conflicts               auto-multiple-choice-devel
 
 } else {
     # devel
-    set amc.version.main        1.5.1
-    set amc.version.secondary   75-ge55d8e1e
-    version                 ${amc.version.main}-${amc.version.secondary}
-    revision                1
-    checksums               rmd160  ff1e07ec51c6ddabda8c410e90033eef5ed51460 \
-                            sha256  d0e4aba705c591adca4a552e31756f6ce1c27a9be1515e8b78896082d393151d \
-                            size    12904898
+    set amc.version.main        1.5.2
+    set amc.version.secondary   20221108151552
+    checksums               rmd160  6531e7e6b41b3580c9e72842cacf703cae527ffd \
+                            sha256  71c2e53a343a87a1aabe4818fb9b9d1ee166fd7d046190d916915640f2d28f86 \
+                            size    13124174
 
     conflicts               auto-multiple-choice
 }
+version                 ${amc.version.main}+git${amc.version.secondary}
```

Comment:
as `port lint` tells you we prefer not to have a `+` character in the `version` field, please just revert to the previous versioning names

Reply:
I updated the port to avoid the lint warning. But the change will prevent the operation of livechek.

Reply:
Sorry to have forgotten the way livecheck works. I updated that part too.Sorry to have forgotten the way livecheck works. I updated that part too.